hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/upgrade/TestUpgradeDowngrade.java

```diff
@@ -557,7 +558,8 @@ public void testDowngradeSixToFiveShouldDeleteRecordIndexPartition() throws Exce
         PARTITION_NAME_COLUMN_STATS,
         PARTITION_NAME_BLOOM_FILTERS,
         PARTITION_NAME_RECORD_INDEX,
-        PARTITION_NAME_FUNCTIONAL_INDEX_PREFIX
+        PARTITION_NAME_FUNCTIONAL_INDEX_PREFIX,
```

Comment:
Should this list be generated from the production code, i.e., list of supported MDT partitions?  Also, do we need to upgrade the table version?  I think master branch is still on table version 6, the same as 0.14.0 release.

Reply:
I think we will eventually need to upgrade the table version for 1.0. But, this PR does not do any incompatible schema changes, so we should be good for now. HUDI-7665 to track.